<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlgEntradaMisionero</class>
 <widget class="QWidget" name="dlgEntradaMisionero">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>494</width>
    <height>321</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Entrada de misioneros</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Misionero</string>
       </property>
       <property name="buddy">
        <cstring>txtPersona</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="myQLineEdit" name="txtPersona"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCheckBox" name="ckLlego">
       <property name="text">
        <string>Llegó</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="ckRegreso">
       <property name="text">
        <string>Regresó</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="ckVolverMirar">
       <property name="text">
        <string>Volver a mirar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Cantidad info</string>
       </property>
       <property name="buddy">
        <cstring>spCantidadInfo</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spCantidadInfo">
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="singleStep">
        <number>5</number>
       </property>
       <property name="value">
        <number>100</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTextEdit" name="txtNotas"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="btFuente">
       <property name="text">
        <string>&amp;Fuente</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btCancelar">
       <property name="text">
        <string>Can&amp;celar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btOK">
       <property name="text">
        <string>&amp;Aceptar</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>myQLineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">widgets/myqlineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
